AlexeyShparun/hometask-4/my_array_rotate.rb

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+# class implementation
+class Array
+  def lab_rotate(value = 1)
+    shift = value % length
+    return self if shift.zero?
+
+    last(length - shift) + first(shift)
+  end
+
+  def lab_rotate!(value = 1)
+    shift = value % length
+    return self if shift.zero?
```

Comment:
since you are re-using `lab_rotate`, you don't need these validations on lines 14-15